java/flight/flight-sql-jdbc-core/src/main/java/org/apache/arrow/driver/jdbc/ArrowDatabaseMetadata.java

```diff
@@ -754,7 +754,11 @@ private <T> T getSqlInfoAndCacheIfCacheIsEmpty(final SqlInfo sqlInfoCommand,
   }
 
   private String convertListSqlInfoToString(final List<?> sqlInfoList) {
-    return sqlInfoList.stream().map(Object::toString).collect(Collectors.joining(", "));
+    if (sqlInfoList == null) {
```

Comment:
Could the items in the list be null? 
Should we explicitly handle it?

Reply:
@vibhatha The values cannot be null.